test/e2e/framework/ci-operator.go

```diff
@@ -116,10 +107,10 @@ func (c *CiOperatorCommand) Run() ([]byte, error) {
 			c.t.Errorf("failed to close logfile: %v", err)
 		}
 	}()
-	log := bytes.Buffer{}
-	tee := io.TeeReader(&log, logFile)
-	c.cmd.Stdout = &log
-	c.cmd.Stderr = &log
+	outputBuffer := bytes.Buffer{}
+	log := io.MultiWriter(logFile, &outputBuffer)
```

Comment:
Why the buffer instead of just os.Stdout?

Reply:
Streaming 100 parallel tests to stdout is not useful. Buffer is needed since we allow tests to check that the output of the command contains things.